src/ui/public/vislib/lib/types/point_series.js

```diff
@@ -130,6 +130,13 @@ export function VislibTypesPointSeries() {
             }
           }
         ];
+      } else {
+        const categoryAxis1 = config.categoryAxes.find((categoryAxis) => {
+          return categoryAxis.id === 'CategoryAxis-1';
+        });
+        if (categoryAxis1) {
+          categoryAxis1.title.text = data.get('xAxisLabel');
```

Comment:
this will always override the title.text defined in configuration with the axis label generated from data ? (